scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/BazelBuildTool.scala

```diff
@@ -12,16 +25,225 @@ class BazelBuildTool(index: IndexCommand) extends BuildTool("Bazel", index) {
     Files.isRegularFile(index.workingDirectory.resolve("WORKSPACE"))
   }
 
+  private def targetSpecs =
+    if (index.buildCommand.isEmpty)
+      List("//...")
+    else
+      index.buildCommand
+
   override def generateScip(): Int = {
-    index
+    val aspectLabel = this.generateAspectFile().getOrElse("")
+    if (aspectLabel.isEmpty) {
+      return 1
+    }
+
+    val scipJavaBinary = index.bazelScipJavaBinary.getOrElse("")
+    if (scipJavaBinary.isEmpty) {
+      index
+        .app
+        .error(
+          "the flag --bazel-scip-java-binary is required to index Bazel codebases. To fix this problem, run scip-java index again with the flag --scip-java-binary=/path/to/scip-java"
+        )
+      return 1
+    }
+
+    val javaHome = index.app.env.environmentVariables.getOrElse("JAVA_HOME", "")
```

Comment:
It's probably a Mac thing, but 

```
~/p/s/scip-java (olafurpg/aspect) > java -version
openjdk version "17.0.6" 2023-01-17
OpenJDK Runtime Environment Temurin-17.0.6+10 (build 17.0.6+10)
OpenJDK 64-Bit Server VM Temurin-17.0.6+10 (build 17.0.6+10, mixed mode)
~/p/s/scip-java (olafurpg/aspect) > echo $JAVA_HOME

```

Reply:
That could be a coursier thing. I use sdkman and it sents JAVA_HOME. It's also configured out of the box on the GH Actions CI runner
```
❯ echo $JAVA_HOME
/Users/olafurpg/.sdkman/candidates/java/current
```

You can fix it by setting the variable with
```
export JAVA_HOME="$(cs java-home)"
```

Reply:
I actually don't use coursier for JDK setup, I use the "standard" way - download a JDK distribution as `.dmg` and install that.

This is a quirk of MacOS which shims `java` executable. Not something that'd happen on a Linux machine.